src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const expencesServices = require('../sevices/expenses.service');
+const userServices = require('../sevices/users.service');
+
+const getAll = (req, res) => {
+  const { userId, from, to, categories } = req.query;
+  let expenses = expencesServices.getAll();
+
+  if (userId) {
+    expenses = expenses
+      .filter(expense => expense.userId === +userId);
```

Comment:
```suggestion
      .filter(expense => expense.userId === Number(userId));
```

The same in other places